lib/models/database/comment/comment_id_firestore.dart

```diff
@@ -0,0 +1,17 @@
+// The comment id is strong typed to avoid misuse
```

Comment:
This comment should not be at the top of the file. It should be a documentation comment on the class, see:
https://github.com/ProximaEPFL/proxima/blob/3a5188de1129ab86c0f4dad909730778dcf75c55/lib/models/database/post/post_id_firestore.dart#L1-L5

Reply:
Completely right, I don't know how I missed it when creating the class.
Fix in 4c0fe6830a9dc7c25e90fb958efa76ed6b5547a5 :+1: